### PR TITLE
Graphics Folder with an implementation of DDA Line Algo in C

### PR DESCRIPTION
**Pull request by rahulkumaran**
Thank you for your contribution. Please provide the details requested below.

## ISSUE NUMBER

This is in reference to the conversation we had on Gitter.

## SHORT DESCRIPTION

This is a program used to implement the DDA Line algorithms and translations in order to make a robot. The translations occur to points as the robot is made around the base point

## TESTING

You just need to make sure you have the graphics.h library before you compile and run this. 
If you already have the library, run the program and the outcome can be seen on the screen

### CODE REVIEW

**Comment by rahulkumaran** [comment]
@prateekiiest please do have a look at it

**Comment by prateekiiest** [comment]
show me the output first

**Comment by rahulkumaran** [comment]
This is the picture of the output @prateekiiestpicture

File: Graphics Algos/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## GRAPHICS ALGORITHMS
+
+This is a folder which contains graphics algorithms and their implementations.
+
+### Robo.c
+
```
Comment (prateekiiest) [inline Graphics Algos/README.md:6]: upload your output pic in the Readme

Comment (rahulkumaran) [reply Graphics Algos/README.md:6]: @prateekiiest added the output pic to the README.md file :)

File: Graphics Algos/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+The robo.c program is a program which helps you in drawing a robot.
+The robot is drawn around a reference point. The DDA Line Algorithm was used in making the program.
+The DDA Line algorithm exists inside the program in a function so you can have a look at what the DDA line algorithm is,
+inside this program itself.<br><br>
```
Comment (prateekiiest) [inline Graphics Algos/README.md:10]: what's this br tag ?

Comment (rahulkumaran) [reply Graphics Algos/README.md:10]: Just to shift the control to the next line. For appearance sake

Comment (rahulkumaran) [reply Graphics Algos/README.md:10]: didn't want everything to be in one para itself. looked too clumsy

Comment (rahulkumaran) [reply Graphics Algos/README.md:10]: I can remove the tags if you want though @prateekiiest

Comment (prateekiiest) [reply Graphics Algos/README.md:10]: ok, that's ok then.

I am merging this 😄

**Review by prateekiiest — APPROVED** [review]

**Pull request merged**